echo-rest/echo-rest.gradle

```diff
@@ -17,9 +17,7 @@
 dependencies {
   implementation "com.squareup.retrofit:retrofit"
   implementation "com.squareup.retrofit:converter-jackson"
-  implementation "com.squareup.okhttp:okhttp"
-  implementation "com.squareup.okhttp:okhttp-urlconnection"
-  implementation "com.squareup.okhttp:okhttp-apache"
+  implementation "com.jakewharton.retrofit:retrofit1-okhttp3-client:1.1.0"
```

Comment:
Seeing this dep leak into multiple PRs. Should be added to spinnaker-dependencies.